modules/services/wob.nix

```
@@ -0,0 +1,75 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib) getExe mkEnableOption mkIf mkOption mkPackageOption optional;
+
+  cfg = config.services.wob;
+  settingsFormat = pkgs.formats.ini { };
+
+  configFile = settingsFormat.generate "wob.ini" cfg.settings;
+in {
+  meta.maintainers = with lib.maintainers; [ Scrumplex ];
+
+  options.services.wob = {
+    enable = mkEnableOption "wob";
+    package = mkPackageOption pkgs "wob" { };
+
+    settings = mkOption {
+      type = settingsFormat.type;
+      description = ''
+        `wob.ini` configuration.
+        See `wob.ini(5)` for documentation
```

```suggestion
        Configuration written to {file}`$XDG_CONFIG_HOME/wob/wob.ini`.
        See {manpage}`wob.ini(5)` for documentation.
```

Also, please place the description field last in the attribute set (after the example).